plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -245,6 +246,18 @@ public boolean isTlsEnabled()
         return "https".equalsIgnoreCase(getControllerUrls().get(0).getScheme());
     }
 
+    public String getProxyRestUri()
```

Comment:
Is an empty or missing value allowed? If so then use an `Optional<String>` instead.